tests/unit/components/buttons/HdRadioButton.spec.js

```diff
@@ -24,24 +23,26 @@ describe('HdRadioButton', () => {
         mobileIcon,
       },
     });
-
-    wrapper.setMethods({ select: mockedSelect });
   });
 
   test('the component is rendered', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
+  test('test the background-image of the radio icon', () => {
+    expect(wrapper.find('.radioButton__icon').attributes().style).toBe('background-image: url(desktop-icon-url);');
```

Comment:
We tend to declare the selectors on top of the file to make it clear what elements the tests depend on and to make it easier to update, because class names are prone to change :)
Example: [HdAlert.spec.js#L4-L5](https://github.com/homeday-de/homeday-blocks/blob/develop/tests/unit/components/HdAlert.spec.js#L4-L5)

Reply:
Done 👍